doc/interfaces/gs_usb.rst

```diff
@@ -8,13 +8,14 @@ and candleLight USB CAN interfaces.
 
 Install: ``pip install "python-can[gs_usb]"``
 
-Usage: pass device ``index`` (starting from 0) if using automatic device detection:
+Usage: pass device ``index`` or ``channel`` (starting from 0) if using automatic device detection:
 
 ::
 
     import can
 
     bus = can.Bus(interface="gs_usb", channel=dev.product, index=0, bitrate=250000)
```

Comment:
Not related to your change, but `dev` is referenced before assignment here. Maybe you could improve this, too.